tests/utils.py

```diff
@@ -37,6 +37,7 @@ def setup_task(
             "binary_image_resolved": binary_image + "-resolved",
             "bundle_mapping": {},
             "index_image": "feed.com/index/image:tag",
+            "index_image_resolved": "feed.com/index/image-resolved:tag",
```

Comment:
would not be better to use some fake URL? or at least fake domain? (just because feed.com is legitimate site)

Reply:
changed